### PR TITLE
Setup Gitpod

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| Documentation PR Link    | 
| License                  | MIT

- Gitpod is like vscode but online (it's like GH codespaces, but already available to everyone).
- It automatically runs `make bootstrap` and `make watch`, so contributors don't have to think about it.*
- It provides 50 hours/month for free on public repositories.
- It pre-builds `master` and PRs.*

Example (this PR): https://gitpod.io/#https://github.com/babel/babel/pull/11704

\* only after merging this PR

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 Contributions are always welcome, no matter how large or small! Before contributing, please read the [code of conduct](https://github.com/babel/babel/blob/master/CODE_OF_CONDUCT.md).
 
+If you want an already configured online IDE to contribute to Babel, you can use Gitpod! [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/babel/babel)
```
Comment (JLHwung) [inline CONTRIBUTING.md:21]: Not sure if it worths a badge. I think a text link is sufficient here.

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# List the start up tasks. You can start them in parallel in multiple terminals. See https://www.gitpod.io/docs/config-start-tasks/
+tasks:
```
Comment (mgower19) [inline .gitpod.yml:2]:
```suggestion
tasks:
```https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod

File: .gitpod.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  - init: make bootstrap
+    command: make watch
+
+github:
+  # https://www.gitpod.io/docs/prebuilds/#configure-prebuilds
```
Comment (mgower19) [inline .gitpod.yml:7]: https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod

File: .gitpod.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# List the start up tasks. You can start them in parallel in multiple terminals. See https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - init: make bootstrap
```
Comment (mgower19) [inline .gitpod.yml:3]: https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod

File: .gitpod.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    branches: true
+    pullRequests: true
+    pullRequestsFromForks: true
+    addCheck: true
+    addComment: false
+    addBadge: true
+    addLabel: false
+
+vscode:
+  extensions:
```
Comment (mgower19) [inline .gitpod.yml:19]: 1.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mgower19) [inline CONTRIBUTING.md:21]: https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod

File: README.md [hunk 6]
```diff
@@ -8,6 +8,9 @@
   The compiler for writing next generation JavaScript.
 </p>
 
+<p align="center">
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/babel/babel)
+</p>
```
Comment (mgower19) [inline README.md:13]: https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod

File: README.md [hunk 7]
```diff
@@ -8,6 +8,9 @@
...
 <p align="center">
     <a href="https://www.npmjs.com/package/@babel/core"><img alt="v7 npm Downloads" src="https://img.shields.io/npm/dm/@babel/core.svg?maxAge=43200&label=v7%20downloads"></a>
   <a href="https://www.npmjs.com/package/babel-core"><img alt="v6 npm Downloads" src="https://img.shields.io/npm/dm/babel-core.svg?maxAge=43200&label=v6%20downloads"></a>
```
Comment (mgower19) [inline README.md:16]: https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod
```suggestion
  <a href="https://www.npmjs.com/package/babel-core"><img alt="v6 npm Downloads" src="https://img.shields.io/npm/dm/babel-core.svg?maxAge=43200&label=v6%20downloads"></a>
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (hzoo) [reply CONTRIBUTING.md:21]: Agreed

Comment (nicolo-ribaudo) [reply CONTRIBUTING.md:21]:
```suggestion
If you want an already configured online IDE to contribute to Babel, you can use [Gitpod](https://gitpod.io/#https://github.com/babel/babel)!
```

Comment (nicolo-ribaudo) [reply CONTRIBUTING.md]: @hzoo I used the badge in the readme because we already have the other badges.

**Pull request merged**